style.css

```diff
@@ -15,20 +15,22 @@
 }
 
 .introduction {
-  width: 20.5rem;
+  display: grid;
+  width: 22.5rem;
   height: 735px;
-  background: #3c3a39;
+  background: #1c1a19;
   margin-top: 10px;
```

Comment:
- [ ] :recycle: As suggested by the previous reviewer there is still room for change in order to make the project look like the the figma design, Kindly put padding-top to zero and adjust(with a width of "100%" related to the main container of "100vw") the other section so that they fill the entire width of the screen:


  - your design:
![SIW](https://user-images.githubusercontent.com/97834160/205415323-e1b05e7a-ac6c-47b4-97be-755e25815ba6.png)

 - figma design:
![fig](https://user-images.githubusercontent.com/97834160/205415381-8992b850-ea8f-485f-9af8-a37c63954c53.png)